extensions/amp-a4a/0.1/test/test-a4a-integration.js

```diff
@@ -107,9 +107,11 @@ describe('integration test: a4a', () => {
         {name: 'ad'}
       );
       adResponse = {
-        headers: {'AMP-Access-Control-Allow-Source-Origin': 'about:srcdoc'},
         body: validCSSAmp.reserialized,
       };
+      if (!adResponse.headers) {
```

Comment:
Will this conditional ever not be true?

Reply:
From the API point of view, it could be. No way for me to know if the implementation may use some particular headers or not. Even with simple CORS there are some whitelisted headers.